src/Controls/src/Core/Device.cs

```diff
@@ -70,72 +65,34 @@ public static void SetFlags(IReadOnlyList<string> flags)
 			Flags = flags;
 		}
 
-		public static void BeginInvokeOnMainThread(Action action)
-		{
-			PlatformServices.BeginInvokeOnMainThread(action);
-		}
+		//[Obsolete("Use BindableObject.Dispatcher instead.")]
```

Comment:
Are these accidental comments or a reminder to obsolete these in the future?

Reply:
They should be added, but I am going to do that in a new PR as it results in hundreds of warings all over the show. I will do a dedicated PR to swap the Device.InvokeOnMain to use the Dispatcher.